utils/funcexp/func_json_length.cpp

```diff
@@ -0,0 +1,110 @@
+#include <cstring>
+#include <string>
+using namespace std;
+
+#include "functor_json.h"
+#include "functioncolumn.h"
+using namespace execplan;
+
+#include "rowgroup.h"
+using namespace rowgroup;
+
+#include "dataconvert.h"
+using namespace dataconvert;
+
+namespace funcexp
+{
+CalpontSystemCatalog::ColType Func_json_length::operationType(FunctionParm& fp,
+                                                              CalpontSystemCatalog::ColType& resultType)
+{
+  if (fp.size() > 1)
+    path.set_constant_flag(fp[2]->data()->getIntVal());
+  return resultType;
+}
+
+int64_t Func_json_length::getIntVal(rowgroup::Row& row, FunctionParm& fp, bool& isNull,
+                                    execplan::CalpontSystemCatalog::ColType& op_ct)
+{
+  const std::string tmp_js = fp[0]->data()->getStrVal(row, isNull);
```

Comment:
This getStrVal returns a ref so I would use ref to avoid std::strin copy-ctor.

Reply:
I change string to std::string_view